hazelcast/reactor.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, dispatcher_map, connection_manager, connection_id, address,
 
     def handle_connect(self):
         self.start_time = time.time()
-        self.logger.debug("Connected to %s", self.connected_address, extra=self._logger_extras)
+        _logger.debug("Connected to %s", self.connected_address)
```

Comment:
nit: the same consideration on debug level check. The same consideration applies to debug logging in other scripts changed by this PR, so I won't duplicate this comment anymore.